tests/base/vectorization_14.output.avx512

```diff
@@ -0,0 +1,57 @@
+
+DEAL::  test 8 array elements
+DEAL::-1.00000 -1.00000 1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000
+DEAL::1.00000 1.00000 -1.00000 1.00000 1.00000 1.00000 1.00000 1.00000
+DEAL::1.00000 1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000
+DEAL::1.00000 1.00000 1.00000 -1.00000 -1.00000 -1.00000 -1.00000 -1.00000
+DEAL::-1.00000 -1.00000 -1.00000 1.00000 1.00000 1.00000 1.00000 1.00000
+DEAL::-1.00000 -1.00000 1.00000 1.00000 -1.00000 -1.00000 -1.00000 -1.00000
```

Comment:
I just ran the test and this is the only output that is wrong (copy-paste issue?):
```suggestion
DEAL::-1.00000 -1.00000 1.00000 1.00000 1.00000 1.00000 1.00000 1.00000
```

Reply:
copy-paste issue. Your version is correct.